client/extensions/woocommerce/state/sites/orders/reducer.js

```diff
@@ -74,6 +97,9 @@ export function items( state = {}, action ) {
 		case WOOCOMMERCE_ORDER_REQUEST_SUCCESS:
 			orders = { [ action.orderId ]: action.order };
 			return Object.assign( {}, state, orders );
+		case WOOCOMMERCE_ORDER_UPDATE_SUCCESS:
```

Comment:
`WOOCOMMERCE_ORDER_UPDATE_SUCCESS` and `WOOCOMMERCE_ORDER_REQUEST_SUCCESS` both do the same thing, so you could save 2 lines here.